app/assets/javascripts/school_administrator_dashboard/StudentsTable.js

```diff
@@ -81,10 +80,6 @@ export default class StudentsTable extends React.Component {
   render() {
     return (
       <div className="StudentsTable" style={styles.root}>
-        <div style={styles.caption}>
-          {this.renderCaption()}
-          <DashResetButton clearSelection={this.props.resetFn} selectedCategory={this.props.selectedCategory}/>
```

Comment:
@edavidsonsawyer Can these props now be removed?  If this component is no longer using these, we should remove them from `propTypes` in this component and then from all the components and tests calling `<StudentsTable />` that are still passing them.

Reply:
Same for `<DashResetButton />`, is it used anywhere else after this change?

Reply:
Yup! Just needed to troubleshoot the escapelistener before I replaced that component.